README.md

```diff
@@ -94,9 +94,35 @@ Browse available modules on
 Standard library modules come from
 [micropython-lib](https://github.com/micropython/micropython-lib) project.
 
-(*) Debian/Ubuntu/Mint derivative Linux distros will require build-essentials,
-libffi-dev and pkg-config packages installed. If you have problems with some
-dependencies, they can be disabled in unix/mpconfigport.mk .
+External dependencies
+---------------------
+
+Building Unix version requires some dependencies installed. For
+Debian/Ubuntu/Mint derivative Linux distros, install `build-essentials`
+(includes toolchain and make), `libffi-dev`, and `pkg-config` packages.
+
+Other dependencies can be built together with MicroPython. Oftentimes,
+you need to do this to enable extra features or capabilities. To build
+these additional dependencies, first fetch git submodules for them:
+
+    $ git submodule update --init
+
+Use this same command to get the latest versions of dependencies, as
+they are updated from time to time. After that:
+
+    $ make deplibs
```

Comment:
This subsection goes straight after instructions for unix version, it starts with "Building Unix version requires ..." and doesn't talk about any other port, so I don't see why it's unclear. Feel free to propose how to improve it, otherwise I already tried to make it both concise and clear.